system-variables.md

```diff
@@ -652,6 +652,17 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
 - Default value: `OFF`
 - This variable is used to set whether the `AUTO_INCREMENT` property of a column is allowed to be removed by executing `ALTER TABLE MODIFY` or `ALTER TABLE CHANGE` statements. It is not allowed by default.
 
+### tidb_analyze_partition_concurrency
```

Comment:
```suggestion
### tidb_analyze_partition_concurrency <span class="version-mark">New in v6.4.0</span>
```

Reply:
I don't think it's a good idea to say it's new in v6.4.0 although it is a fact. The feature owner wants to make it public in a future release. If we say it's new in v6.4.0, it will sound like the related unreleased feature is new in v6.4.0.